release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2763,6 +2763,11 @@ $ oc adm release info 4.15.25 --pullspecs
 
 * Previously, when you created or deleted large volumes of service objects simultaneously, the service controller's ability to process each service sequentially would slow down. This caused short timeout issues for the service controller and backlog issues for the objects. With this release, the service controller can now process up to 10 service objects simultaneously to reduce the backlog and timeout issues. (link:https://issues.redhat.com/browse/OCPBUGS-36821[*OCPBUGS-36821*])
 
+[id="ocp-4-15-25-known-issue_{context}"]
+==== Known issues
+
+* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF fail to create a pod sandbox and do not function. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
```

Comment:
OCPBUGS-38091 is to track the fix for 4.14.z
OCPBUGS-38090 tracks fix for 4.15.z